src/__tests__/TranslationDispatcher.test.ts

```diff
@@ -0,0 +1,116 @@
+import { DOMNodesTranslator } from '../DOMNodesTranslator';
+import { NodesIntersectionObserver } from '../NodesIntersectionObserver';
+import { TranslationDispatcher } from '../TranslationDispatcher';
+import { configureTranslatableNodePredicate } from '../utils/nodes';
+import { awaitTranslation, containsRegex, TRANSLATION_SYMBOL, translator } from './utils';
+
+require('intersection-observer');
+
+beforeEach(() => {
+	vi.clearAllMocks();
+	document.body.innerHTML = '';
+});
+
+const isTranslatableNode = () => true;
+
+test('Translate node that is not suitable for delayed translation', async () => {
```

Comment:
Add context in description. This test is about "Translation in lazy translation mode ..."